plugin/src/create-schema-customization.ts

```diff
@@ -0,0 +1,228 @@
+import { CreateSchemaCustomizationArgs } from "gatsby";
+
+export function createSchemaCustomization(
+  { actions, schema }: CreateSchemaCustomizationArgs,
+  pluginOptions: ShopifyPluginOptions
+) {
+  const includeCollections =
+    pluginOptions.shopifyConnections &&
```

Comment:
Could potentially do this in a different way:
`const includeCollections = pluginOptions.shopifyConnections?.includes("collections")`